nemo/collections/nlp/models/token_classification/punctuation_capitalization_model.py

```diff
@@ -246,18 +245,20 @@ def _setup_dataloader_from_config(self, cfg: DictConfig):
             tokenizer=self.tokenizer,
             text_file=text_file,
             label_file=label_file,
-            pad_label=self.model_cfg.pad_label,
+            pad_label=self._cfg.dataset.pad_label,
             punct_label_ids=self.punct_label_ids,
             capit_label_ids=self.capit_label_ids,
-            max_seq_length=self.model_cfg.max_seq_length,
-            ignore_extra_tokens=self.model_cfg.ignore_extra_tokens,
-            ignore_start_end=self.model_cfg.ignore_start_end,
-            use_cache=self.model_cfg.use_cache,
+            max_seq_length=self._cfg.dataset.max_seq_length,
+            ignore_extra_tokens=self._cfg.dataset.ignore_extra_tokens,
+            ignore_start_end=self._cfg.dataset.ignore_start_end,
+            use_cache=self._cfg.dataset.use_cache,
             num_samples=cfg.num_samples,
         )
         if cfg.prefix == 'train':
             self.punct_label_ids = dataset.punct_label_ids
             self.capit_label_ids = dataset.capit_label_ids
+            self.register_artifact('punct_label_ids.csv', punct_label_ids_file)
```

Comment:
what does this do?

Reply:
it saves a file for label name to lable_id conversion needed for Jarvis